astro/src/content/quickstarts/quickstart-javascript-react-web.mdx

```diff
@@ -0,0 +1,198 @@
+
+#### Create a Home Page
+
+The next step is to get a basic home page up and running. We’ll take this opportunity to copy in all the images and CSS style files that you’ll need for the application.
+
+Run the following copy commands to copy these files from the quickstart repo into your project. This assumes that you checked the quickstart repo out into the parent directory. If that’s not the case, replace the .. below with your actual repo location.
+
+```shell
+cp -r ../complete-application/src/assets src && \
+cp -r ../complete-application/src/index.css src
+```
+
+The home page will be a simple page with a welcome message and a login link. Create a new directory `pages` under `src`, in that directory create a new file `src/pages/HomePage.js`:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-react-web/main/complete-application/src/pages/HomePage.js"
+            lang="jsx"/>
+
+#### Create an Account Page
+
+The account page will be a simple page that displays a random balance for the logged in user. Create a new file `src/pages/AccountPage.js`:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-react-web/main/complete-application/src/pages/AccountPage.js"
+            lang="jsx"/>
+
+#### Create A Make Change Page
+
+The make change page will be a simple page that displays an input field for the user to enter a dollar amount and a button to convert that amount into coins. Create a new file `src/pages/MakeChangePage.js`:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-react-web/main/complete-application/src/pages/MakeChangePage.js"
+            lang="jsx"/>
+
+## Authentication
+
+You now have created a basic React application with a home page and an account page.
+
+Depending on the user's authentication state, the login or logout button should be displayed in the header. For this create a new directory `components` in `src` and a new file `src/components/LogoHeader.js`:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-react-web/main/complete-application/src/components/LogoHeader.js"
+            lang="jsx"/>
+
+Additionally, we want to display different menu items. For this create a new file `src/components/MenuBar.js`:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-react-web/main/complete-application/src/components/MenuBar.js"
+            lang="jsx"/>
+
+The next step is to tie it all together. Update the `src/App.js` file to add the routes and the header:
+
+<RemoteCode url="https://raw.githubusercontent.com/sonderformat-llc/fusionauth-quickstart-javascript-react-web/main/complete-application/src/App.js"
+            lang="jsx"/>
+
+## Running the Application
+
+You can now run the application with the following command:
+
+```shell
+npm start
```

Comment:
The problem is that the astro of fusionauth-site runs per default on port 3000, therefore I usually start our astro with `npm run dev -- --port 3001` to have the site open at the same time while doing the quickstart examples. Maybe it would be a good idea to change the fusionauth-site astro port to something not used that common as port 3000.

I'll add a "aside note" to the guide anyway.

@mooreds

Reply:
I'm not sure what your question is, @escii ? I think you're right that we shouldn't treat this special and that we should trust that people know how to run their chosen framework on a port that isn't currently used. Let's drop it, no need for an aside.